README.md

```diff
@@ -1,68 +1,38 @@
-
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
-
-You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
-
-## Learn More
-
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
-
-To learn React, check out the [React documentation](https://reactjs.org/).
-
-### Code Splitting
-
-This section has moved here: https://facebook.github.io/create-react-app/docs/code-splitting
-
-### Analyzing the Bundle Size
-
-This section has moved here: https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size
-
-### Making a Progressive Web App
-
-This section has moved here: https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app
-
-### Advanced Configuration
-
-This section has moved here: https://facebook.github.io/create-react-app/docs/advanced-configuration
-
-### Deployment
-
-This section has moved here: https://facebook.github.io/create-react-app/docs/deployment
-
-### `yarn build` fails to minify
-
-This section has moved here: https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify
+<h1 align="center">If You Can't Walk</h1>
+<br/>
+<p align="center">
+<img src="https://s3.amazonaws.com/libapps/accounts/126774/images/Black_Power_logo.jpg" alt="blm" width="500px" ></img>
+</p>
+<br/>
+<h1 align=center>What is it?</h1>
+<p >This platform is a guide for those who would like to help the [BlackLivesMatter](https://blacklivesmatter.com/) community in the fight for social injustice. As Black people in the United States we face many cases of racial discrimination to a day to day basis, and we'd like to take a stand. Help make a change in history by first using this platform to help those in need of the revolution.  </p>
+<p align="center">Empowered by the inspirational quote by Dr.Martin Luther King Jr:</p>
+
+>“If you can't fly then run, if you can't run then walk, **if you can't walk** then crawl, but whatever you do you have to keep moving forward.”
+
+
+<h1 align="center">Creators</h1>
+<p align="center">
+<img src="https://avatars2.githubusercontent.com/u/46658928?s=400&u=e6781c56b75fe49ca2088d1e6e4af5b8399e1e11&v=4" alt="MO.png" >
+</p>
+<h2 align="center">Mikayla Orange</h2>
+<div align="center">
+ • Computer Science student at Howard University
+
+ • Google Tech Exchange Scholar
+ 
+ •[Check out my website](https://mikaylaorange.com/)
```

Comment:
My site is not ready yet king lmao so you might wanna remove that or link to my LinkedIn instead

Reply:
Overall, this is really good! Thank you!